src/nativeUI.js

```diff
@@ -1,5 +1,35 @@
-import { ipcRenderer } from "electron";
+const { ipcRenderer, dialog } = require("electron");
 
-exports.promptUserToSaveContentToFile = function(contents) {
+const promptUserToSaveContentToFile = contents => {
   ipcRenderer.send("saveValueToFile", contents);
 };
+
+const getFileNameFromUser = () => {
+  return dialog.showSaveDialog({
```

Comment:
I'm surprised this is not async. Does it just block until it gets a result?

Reply:
Now it does block. Previously, it was async, but that allowed the user to open multiple save dialogs.